src/services/bib-bulk.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-warning-comments */
+
+/**
+*
+* @licstart  The following is the entire license notice for the JavaScript code in this file.
+*
+* RESTful API for Melinda
+*
+* Copyright (C) 2018-2019 University Of Helsinki (The National Library Of Finland)
+*
+* This file is part of melinda-rest-api
+*
+* melinda-rest-api program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* melinda-rest-api is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU Affero General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*
+* @licend  The above is the entire license notice
+* for the JavaScript code in this file.
+*
+*/
+
+import {Utils} from '@natlibfi/melinda-commons';
+import amqplib from 'amqplib';
+import {AMQP_URL} from '../config';
+import {logError} from '../utils';
+
+const {createLogger} = Utils;
+const logger = createLogger();
+
+export async function pushToQueue({queue, QUEUEID, format, records, operation}) {
```

Comment:
The records should be parsed with marc-record-serializers because we need to support multiple formats. Also, this application can now be MARC format agnostic.  The importer application needs to send only Aleph sequential forward to aleph-record-load-api (Which it parses from MARC JSON from the queue).